package.json

```diff
@@ -17,6 +17,7 @@
     "create-hmac": "^1.1.7",
     "crypto-js": "^4.0.0",
     "js-sha3": "^0.8.0",
+    "symbol-sdk": "^1.0.1",
```

Comment:
@aleixmorgadas . The HD library doesn't depend on the SDK on production, only for tests. This way, we keep the library small and release independent.  Could you move the dependency back to dev?

Reply:
missed that part, done 👍 😄